src/main/java/com/launchdarkly/client/RedisFeatureStoreConfig.java

```diff
@@ -0,0 +1,205 @@
+package com.launchdarkly.client;
+
+import org.apache.http.client.utils.URIBuilder;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class exposes advanced configuration options for the {@link com.launchdarkly.client.RedisFeatureStore}.
+ * Instances of this class must be constructed with a {@link Builder}
+ */
+public class RedisFeatureStoreConfig {
```

Comment:
Okay this has now been updated to be a `RedisFeatureStoreBuilder` where `.build()` returns a valid `RedisFeatureStore`